PiwikTracker.php

```diff
@@ -1583,6 +1591,8 @@ protected function sendRequest($url, $method = 'GET', $data = null, $force = fal
             if (!empty($response)) {
                 list($header, $content) = explode("\r\n\r\n", $response, $limitCount = 2);
             }
+            
+            $this->parseIncomingCookies(explode("\r\n", $header));
```

Comment:
When I tested the PR I had a case where `$header` was not defined because I received an empty response. I would suggest to check whether `$header` is set by wrapping the call in `if (!empty($header)) { ... }`

Reply:
I just realize we will then also need an `else { $this->incomingTrackerCookies = array(); }`

Reply:
@MichaelHeerklotz I think here is still a 
```
if (!empty($header)) {
    $this->parseIncomingCookies(explode("\r\n", $header));
} else {
    $this->incomingTrackerCookies = array();
}
```

missing but we can patch this also after merging.

Reply:
I have added a $header = ''; , so it should be fine, or am I missing something?

Reply:
Sorry I did not see that. Stupid me, that's a much better solution! Cheers :)